app/src/main/java/net/skyscanner/backpack/demo/stories/IconsAdapter.kt

```diff
@@ -51,6 +51,10 @@ class IconsAdapter(
     holder.img.setOnClickListener {
       BpkToast.makeText(holder.itemView.context, names[position], BpkToast.LENGTH_SHORT).show()
     }
+    holder.img.contentDescription = names[position]
```

Comment:
announce it in a nice format - clicking it still announces the full resource name